lib/text.rb

```diff
@@ -0,0 +1,21 @@
+require 'twilio-ruby'
+
+class Text
+
+  def confirmation
+
+    account_sid = 'ACd7bd438097782e4c79a158fb3500290e'
```

Comment:
No use of ENV (Environment Variables). I would look at removing this information from GitHub and using Environment Variables instead. Try asking your cohort to see if anyone has an example of doing this.